config/locales/en.yml

```diff
@@ -1320,40 +1320,40 @@ en:
-              no_date_header: Date QTS or EYTS standards met is required
-              no_dates_given: No dates have been provided in this CSV
+              no_id_header: The selected file must contain a column for TRN, trainee provider ID or HESA ID
+              no_date_header: The selected file must contain a column for date QTS or EYTS standards met
+              no_dates_given: The selected file must contain date QTS or EYTS standards met for at least one trainee
               no_trainees: The selected file must contain at least one trainee
         bulk_update:
           recommendations_uploads:
             validate_trainee:
-              multiple_trainees_trn_recieved: Multiple trainees in state 'trn received' found via %{found_with}
-              multiple_trainees_found_via: Multiple trainees found via %{found_with}
-              trainee_wrong_state: Trainee is %{state}
-              no_trainee_matched: No trainee could be matched
+              multiple_trainees_trn_recieved: %{count} trainee records with TRN received status have the same %{found_with} - contact becomingateacher@digital.education.gov.uk to fix this
+              multiple_trainees_found_via: %{count} trainee records have the same %{found_with} but none have TRN received status - you can only recommend trainees with TRN received status
+              trainee_wrong_state: The trainee has %{state} status - you can only recommend trainees with TRN received status
+              no_trainee_matched: %{id_available} must match a trainee record
             validate_csv_row:
-              trn_format: TRN must be 7 characters long and contain only numbers
-              hesa_id_format: HESA ID must be 13 or 17 characters long and contain only numbers
-              award_date_future: Award date must not be in the future (%{date})
-              award_date_past: Award date cannot be from more than 12 months ago (%{date})
-              date_standards_met: Date standards met cannot be before the ITT start date (%{date})
-              date_parse: Date could not be parsed, please use the format dd/mm/yyyy e.g. 27/02/2022
-              trn: Trainee TRN does not match
-              hesa_id: Trainee HESA ID does not match
-              provider_trainee_id: Provider trainee id does not match
-              first_names: Trainee first names do not match
-              last_names: Trainee last names do not match
-              lead_school: Lead school does not match
-              qts_or_eyts: QTS/EYTS declaration does not match
-              route: Route does not match
-              phase: Phase does not match
-              age_range: Age range does not match
-              subject: Subject does not match
+              trn_format: TRN must be 7 numbers
+              hesa_id_format: HESA ID must be 13 or 17 numbers
+              award_date_future: Date QTS or EYTS standards met must be today or in the past
```

Comment:
@johndoates didn't we talk about saying the date cannot be in the future?

Reply:
We did but I've used the standard wording from the [design system](https://design-system.service.gov.uk/components/date-input/#error-messages).